### DISCUSSION TITLE
UI/UX idea for anime entry

### DISCUSSION THREAD

**Discussion by ghost** [imashnake0/Animite#101] [Show and tell]
Hi, I was doing some UI/UX examples in my own, I put it here in case you want to take a look https://www.figma.com/file/3sQWwouruYBt8GyFlllWm6/Animite

Of course I understand if you have other idea and I understand too if you don't want to implement it, I only share it to give you some ideas or references.

**Comment by imashnake0** [comment]
These look great! I shall make use of them when I work on the tabs.
